lockdown.go

```diff
@@ -46,109 +44,129 @@ type ZoneLockdownListResponse struct {
 	ResultInfo `json:"result_info"`
 }
 
+type LockdownListParams struct {
+	PerPage uint16
+	Page    uint16
+	ResultInfo
```

Comment:
`ResultInfo` already has pagination structs embedded so this isn't required - https://github.com/cloudflare/cloudflare-go/blob/6fc060e04d39e4eda023336e4854871352a1edf4/cloudflare.go#L427-L435